src/hooks/useHasInstalledApp.ts

```diff
@@ -0,0 +1,23 @@
+import {useEffect, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+const useHasInstalledApp = () => {
+    const [lastIOSLogin] = useOnyx(ONYXKEYS.NVP_LAST_ECASH_IOS_LOGIN, {canBeMissing: true});
```

Comment:
@nyomanjyotisa We should use `const [lastIOSLogin, lastIOSLoginResult] = useOnyx(ONYXKEYS.NVP_LAST_ECASH_IOS_LOGIN, {canBeMissing: true});` and get rid of the setTimeout.